GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -372,45 +288,27 @@ public void run() {
      * those that use lots of memory, like large textures) that you
      * <em>know</em> are no longer being used.
      * 
-     * @since 2.0.0
+     * @since 3.0.0
      */
-    @Override
-    public final void close() throws IOException {
-        synchronized (sReferenceSet) {
-            if (mNativePointer != 0L) {
-                GVRReference reference = findReference(mNativePointer);
-                if (reference != null) {
-                    reference.close();
-                    mNativePointer = 0L;
-                }
-            }
-        }
+    public final void releaseNative() {
+        mGVRContext.releaseNative(this);
```

Comment:
it's better to modularize the GVRContext now while adding new API. but this is totally optional.